app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
```

Comment:
đưa cái này vào model user và sửa những chỗ dùng đến nhé

Reply:
anh ơi đưa hết hay đưa cái nào ạ!

Reply:
anh ơi em có đưa current_user và đưa vào model User rồi, nhưng em chưa biết cách làm thế nào để gọi một method trong model để sử dụng trong helpers. Hix hix. Search trên GG thì nó toàn hướng dẫn theo chiều ngược lại ạ. Anh chỉ giáo em với.!

Reply:
@tientt-glose gọi thông qua instance của model đó 
User thì dùng user.method nhé